src/ElfUtils/testdata/Makefile

```diff
@@ -8,8 +8,6 @@
 CC=clang
 PHONY=all clean
 
-all: line_info_test_binary line_info_test_binary_compressed test_library
-
```

Comment:
This was probably not intended, was it?

Reply:
it was, I figured in most cases we do not want to compile every single test library, and muscle memory is not really helping here.

Reply:
Okay. Makes sense. But in that case you should also remove `all` from the list of phony targets.

Reply:
done.